src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -68,22 +70,54 @@
 
     private boolean sessionEnabled = true;
 
-    @NotNull
```

Comment:
In fact, we can now define authorization generators via the `configClass` option, so it becomes optional to directly set the  `authorizationGenerators`.